server/plugin/api.go

```diff
@@ -124,6 +125,32 @@ func (p *Plugin) handleLink(w http.ResponseWriter, r *http.Request) {
 	w.Header().Add("Content-Type", "application/json")
 }
 
+// handleGetAllLinkedProjects returns all linked projects list
+func (p *Plugin) handleGetAllLinkedProjects(w http.ResponseWriter, r *http.Request) {
+	mattermostUserID := r.Header.Get(constants.HeaderMattermostUserIDAPI)
+	projectList, err := p.Store.GetAllProjects(mattermostUserID)
```

Comment:
If this function `GetAllProjects` is fetching data from Azure API, then it should also return the status code.

Reply:
its actually extracting data from the KV store